week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,31 +1,36 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getWordLength(word) {
-  return "word".length()
+  return word.length;
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  return a * b * c;
 }
 
 /* ======= TESTS - DO NOT MODIFY ===== */
 
 function test(test_name, expr) {
   let status;
   if (expr) {
-      status = "PASSED"
+    status = "PASSED";
   } else {
-      status = "FAILED"
+    status = "FAILED";
   }
 
-  console.log(`${test_name}: ${status}`)
+  console.log(`${test_name}: ${status}`);
 }
 
-test("fixed trimWord function", trimWord("  CodeYourFuture ") === "CodeYourFuture")
-test("fixed wordLength function", getWordLength("A wild sentence appeared!") === 25)
-test("fixed multiply function", multiply(2,3,6) === 36)
\ No newline at end of file
+test(
+  "fixed trimWord function",
+  trimWord("  CodeYourFuture ") === "CodeYourFuture"
+);
+test(
+  "fixed wordLength function",
+  getWordLength("A wild sentence appeared!") === 25
+);
+test("fixed multiply function", multiply(2, 3, 6) === 36);
```

Comment:
Good one. The main concept here is to differentiate between methods and properties. (trim is a method, length is a property of string )